src/addComponentBoundary.coffee

```diff
@@ -1,4 +0,0 @@
-import addWrapper from './addWrapper'
```

Comment:
Unrelated, it wasn't documented but fully removed `addComponentBoundary()`

It's just a trivial wrapper around `addWrapper()` and doesn't belong in ad-hok (maybe in an ad-hok utils package)